src/app/pages/datasets/components/dataset-capacity-management-card/dataset-capacity-management-card.component.ts

```diff
@@ -32,24 +66,40 @@ export class DatasetCapacityManagementCardComponent implements OnChanges {
   ) {}
 
   ngOnChanges(): void {
-    if (!this.dataset.locked) {
+    this.loadExtraProperties();
+    if (this.checkQuotas) {
       this.getQuotas();
     }
   }
 
+  loadExtraProperties(): void {
+    this.cdr.markForCheck();
+    this.extraPropertiesSubscription?.unsubscribe();
+    this.extraPropertiesSubscription = this.ws.call('pool.dataset.query', [[['id', '=', this.dataset.id]]]).pipe(
+      map((datasets) => datasets[0]),
+      untilDestroyed(this),
+    ).subscribe((dataset) => {
+      this.extraProperties = dataset;
+      this.cdr.markForCheck();
+    });
+  }
+
   getQuotas(): void {
-    this.loading = true;
+    this.isLoading = true;
     this.cdr.markForCheck();
     this.quotasSubscription?.unsubscribe();
     this.quotasSubscription = forkJoin([
       this.ws.call('pool.dataset.get_quota', [this.dataset.id, DatasetQuotaType.User, []]),
       this.ws.call('pool.dataset.get_quota', [this.dataset.id, DatasetQuotaType.Group, []]),
+      this.ws.call('pool.dataset.get_quota', [this.dataset.id, DatasetQuotaType.Dataset, []]),
```

Comment:
Does this actually work? I was under impression that you'd have to traverse the topology tree to find the parent with the quota.